### PR TITLE
Update wdio-mediawiki to 1.1.0 and mwbot to 2.0

### PR DESCRIPTION
**Pull request by kostajh**
See also https://gerrit.wikimedia.org/r/c/mediawiki/core/+/680336

Bug: T280332

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
 		"test": "eslint ."
 	},
 	"peerDependencies": {
-		"mwbot": "^1.0.10",
+		"mwbot": "^2.0.0",
 		"wdio-mediawiki": "^1.0.0",
```
Comment (jakobw) [inline package.json]: The latest published release of [wdio-mediawiki](https://www.npmjs.com/package/wdio-mediawiki) still depends on mwbot v1 so these two peer dependencies are conflicting at the moment. I see that the update already happened in the code ([here](https://gerrit.wikimedia.org/r/c/mediawiki/core/+/680336)) but we would need a new version to be released before we can use it.

**Review by jakobw — CHANGES_REQUESTED** [review]
Thanks for the update! I tested it locally and it all works fine, aside from the the peer dependency problem I mentioned in the comment. As soon as the new version wdio-mediawiki is published and updated here as well, we can merge this.

File: package.json (hunk 1 above)
Comment (kostajh) [reply package.json]: Good point :) I've made https://gerrit.wikimedia.org/r/c/mediawiki/core/+/693400, after that's merged @zeljkofilipin could you publish the new package version on npmjs.com?

**Comment by kostajh** [comment]
New version is published: https://www.npmjs.com/package/wdio-mediawiki

**Comment by kostajh** [comment]
btw, I don't think it's really necessary to specify both wdio-mediawiki and mwbot as peerDependencies, should we just list `wdio-mediawiki` and let that pull in the correct version of mwbot?

**Comment by jakobw** [comment]
> btw, I don't think it's really necessary to specify both wdio-mediawiki and mwbot as peerDependencies, should we just list wdio-mediawiki and let that pull in the correct version of mwbot?

It's true that this would work in the current setup, however, I believe the http/mw api client (`mwbot` in this case) should be an implementation detail of `wdio-mediawiki`. `wdio-wikibase` [depends on `mwbot`](https://github.com/wmde/wdio-wikibase/blob/a39eaff4578f929c8fe027a3e0e9810ad2f5655e/wikibase.api.js#L3) directly, which means that if we don't explicitly list it in its `package.json`, `wdio-wikibase` is going to break in a hypothetical scenario in which `wdio-mediawiki` switches from `mwbot` to `next-shinier-thing`.

**Review by jakobw — APPROVED** [review]
LGTM and verified locally! Thanks for updating this and for poking people to get `wdio-mediawiki` released! :)

**Pull request merged**